Makefile

```diff
@@ -1,29 +1,36 @@
-ASSETS := $(shell yq e '.assets.[].src' manifest.yaml)
-VERSION := $(shell yq e ".version" manifest.yaml)
 IPFS_SRC := $(shell find ./go-ipfs)
-S9PK_PATH=$(shell find . -name ipfs.s9pk -print)
+PKG_VERSION := $(shell yq e ".version" manifest.yaml)
+PKG_ID := $(shell yq e ".id" manifest.yaml)
 TS_FILES := $(shell find . -name \*.ts )
 
 # delete the target of a rule if it has changed and its recipe exits with a nonzero exit status
 .DELETE_ON_ERROR:
 
 all: verify
 
+clean:
+	rm -rf docker-images
+	rm -f  $(PKG_ID).s9pk
+	rm -f image.tar
+	rm -f scripts/*.js
+
 install: all
 	embassy-cli package install ipfs.s9pk
 
-verify: ipfs.s9pk $(S9PK_PATH)
-	embassy-sdk verify s9pk $(S9PK_PATH)
+verify: $(PKG_ID).s9pk
+	embassy-sdk verify s9pk $(PKG_ID).s9pk
 
-clean:
-	rm -f image.tar
-	rm -f ipfs.s9pk
-
-ipfs.s9pk: manifest.yaml image.tar instructions.md scripts/embassy.js
+$(PKG_ID).s9pk: manifest.yaml instructions.md scripts/embassy.js LICENSE docker-images/aarch64.tar docker-images/x86_64.tar
+	if ! [ -z "$(ARCH)" ]; then cp docker-images/$(ARCH).tar image.tar; fi
```

Comment:
Lucy mentioned we dont need this line anymore if you are building a fattie 
`if ! [ -z "$(ARCH)" ]; then cp docker-images/$(ARCH).tar image.tar; fi`

Reply:
removed